requirements-test.txt

```diff
@@ -1,2 +1,4 @@
 coveralls
+django-redis
 openwisp-utils[qa]>=0.4.1
+redis
```

Comment:
I think this should move to the `requirements.txt` file as celery depends on it in a working environment

Reply:
But we ask users to choose their own celery broker. Adding `redis` to `requirements.txt` will somehow bind them onto using Redis. That's why I added these in `requirements-test.txt` as it has been done in [openwisp-monitoring](https://github.com/openwisp/openwisp-monitoring/blob/master/requirements-test.txt).

Reply:
Yes @pandafy. That's a nice way of thinking. But let's know that we need to have default broker with which celery will work with when this code gets out there when deployed. The `requirements-test.txt` is not run in production. So while running your `requirements.txt` you should get everything you need to get your module up and running without any configuration. The tech gig will have the experience to get in and change their broker. You've done well by including the settings in the readme.

Reply:
Thank you @NoumbissiValere , I have made those changes. :smile:

Reply:
I'm sorry I was not fast to reply to this, but @NoumbissiValere you are not right regarding to this, @pandafy was following what we already do in other modules.
The fact that celery uses redis, is a deployment choice, and not an application choice. Celery can be deployed using many different brokers (RabbitMQ, memcached, database) and we should not force the usage of redis.
Please keep in mind for the next time.

Reply:
Thank you @nemesisdesign and sorry @pandafy :smile: